x/hub-genesis/keeper/genesis.go

```diff
@@ -9,8 +12,20 @@ import (
 func (k Keeper) InitGenesis(ctx sdk.Context, genState *types.GenesisState) {
 	k.SetParams(ctx, genState.Params)
 	k.SetState(ctx, genState.State)
-	for _, seq := range genState.UnackedTransferSeqNums {
-		k.saveUnackedTransferSeqNum(ctx, seq)
+
+	err := k.PopulateGenesisInfo(ctx)
+	if err != nil {
+		panic(fmt.Sprintf("generate genesis info: %s", err))
+	}
+
+	// validate the funds in the module account are equal to the sum of the funds in the genesis accounts
+	expectedTotal := math.ZeroInt()
```

Comment:
do we check somewhere the maximun number of genesis accounts allowed?

Reply:
on the hub side